src/obr/signac_wrapper/operations.py

```diff
@@ -55,6 +67,40 @@ def set_entrypoint(self, entrypoint: dict):
         """
         self._entrypoint = entrypoint
 
+    def group_jobs(self, jobs: list[Job], summarize: int = 0) -> dict[str, list[Job]]:
+        """Returns the list of jobs of the given OpenFOAMProject where the last `summarize` levels are grouped together at the corresponding parent view."""
+        grouped = {}
+        id_view_map = map_view_folder_to_job_id(os.path.join(self.path, "view"))
```

Comment:
I think we should pass it as parameter, then we are more flexible.

Reply:
Just to be clear, you suggest to pass the path, or the view map?

Reply:
Im going to go ahead and assume the former

Reply:
no the latter, generate the view map outside of this function and pass the map to group_jobs. Because we might pregenerate the map and use it somewhere else too.